README.md

```diff
@@ -30,6 +30,12 @@ OpenSearch and its components are built from source, assembled, signed and teste
 
 TODO
 
+#### Releasing for FreeBSD
+
+The FreeBSD ports and packages for OpenSearch are managed by an *[OpenSearch Team](https://wiki.freebsd.org/OpenSearch)* at FreeBSD.  When a new release is rolled out, this team is in charge for updating the port and committing it to the FreeBSD ports tree. Anybody is welcome to help the team by providing patches for [upgrading the ports](https://docs.freebsd.org/en/books/porters-handbook/book/#port-upgrading) following the [FreeBSD Porter's Handbook](https://docs.freebsd.org/en/books/porters-handbook/book/) instructions.
```

Comment:
Let's say "by a community OpenSearch team", we want to give credit to the fact that it's not maintained by an amazon team

Reply:
Done!